fft/src/KokkosFFT_Cuda_transform.hpp

```diff
@@ -10,45 +10,47 @@
 
 namespace KokkosFFT {
 namespace Impl {
-template <typename... Args>
-inline void exec_plan(cufftHandle& plan, cufftReal* idata, cufftComplex* odata,
-                      int /*direction*/, Args...) {
-  cufftResult cufft_rt = cufftExecR2C(plan, idata, odata);
+
+struct ScopedCufftPlan;
```

Comment:
I feel we should include `KokkosFFT_Cuda_type.hpp` instead because the code below requires the complete definition of the `ScopedCufftPlan` ?

Reply:
Sure